mmv1/products/bigquerydatatransfer/Config.yaml

```diff
@@ -222,10 +223,28 @@ properties:
       to a different credential configuration in the config will require an apply to update state.
     url_param_only: true
     properties:
+      - name: 'secretAccessKeyWoVersion'
+        type: Integer
+        immutable: true
```

Comment:
This is wrong, right? Otherwise the WO variant is immutable while the sensitive one isn't.

Reply:
yeah this is wrong and addressed here: https://github.com/GoogleCloudPlatform/magic-modules/pull/13135/commits/31f7b4ee0e49430c1d8f26bf8f4c1ec3bd23b91c